test-output/Default suite/Default test.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If the .gitignore file mentioned above doesn't ignore the test-output directory, it should be added. Artifacts of a build (test output, classes, reports, compiled jars, etc.) should not be stored in git